src/log.js

```diff
@@ -481,6 +516,52 @@ export class Log {
       error.message = error.message.replace(USER_ERROR_SENTINEL, '');
     }
   }
+
+  /**
+   * @param {!Array} args
+   * @return {!Array}
+   * @private
+   */
+  maybeExpandArguments_(args) {
+    if (isArray(args[0])) {
+      return [this.expandLogMessage_(/** @type {!Array} */ (args[0]))];
+    }
+    return args;
+  }
+
+  /**
+   * Either redirects a pair of (errorId, ...args) to a URL where the full
+   * message is displayed, or displays it from a fetched table.
+   *
+   * This method is used by the output of the `transform-log-methods` babel
+   * plugin. It should not be used directly. Use the (*error|assert*|info|warn)
+   * methods instead.
+   *
+   * @param {!Array} parts
+   * @return {string}
+   * @private
+   */
+  expandLogMessage_(parts) {
+    const id = this.assert(parts[0]);
```

Comment:
Can you explain this. We're passing in the message as the `shouldBeTruish` param to assert?

Reply:
Where? `shouldBeTruish` stays in place.

Reply:
We're calling assert here, and the first param is `shouldBeTruish`. I don't understand why we need to call assert with a message that we know is there.

Reply:
Ah, I see what you mean now.